docs/content/oracleobjectstorage.md

```diff
@@ -12,6 +12,10 @@ versionIntroduced: "v1.60"
 Paths are specified as `remote:bucket` (or `remote:` for the `lsd` command.)  You may put subdirectories in 
 too, e.g. `remote:bucket/path/to/dir`.
 
+## Tutorials
+
+- [Mount buckets as local file system](../../backend/oracleobjectstorage/docs/oos-mount.md)
```

Comment:
This link won't work on the website as we don't expose anything from backend.

It needs to go in docs/content

I think (and I haven't tried this!) you can turn `docs/content/oracleobjectstorage.md` into a directory `docs/content/oracleobjectstorage/index.md` then that gives you an obvious place to put the extra docs. I don't think this changes the URL of the main docs page.

Reply:
@ncw thank you for surfacing the docs bug. I have fixed it.
- moved the tutorials to docs/content